pkg/okteto/context.go

```diff
@@ -467,5 +473,12 @@ func (okctx *OktetoContext) ToUser() *User {
 		GlobalNamespace: okctx.GlobalNamespace,
 	}
 	return u
+}
 
+func GetTelemetryEnabled() bool {
+	t, err := strconv.ParseBool(Context().TelemetryEnabled)
```

Comment:
For clusters not managed by okteto, telemetry should be always `true`

Reply:
Changed this to IsTelemetryEnabled where the okteto cluster is checked and true is return in case is not